<log>
  <summary verdicts='2' failures='2' errors='0'/>
  <section name='Test' method='verdict_refute_nil?' type='fail'>
    <section name='Args=[nil]'>
      <verdict method='verdict_refute_nil?' outcome='failed' id='0' message='Message 0'>
        <actual class='NilClass' value='nil'/>
        <exception class='Minitest::Assertion' message='Expected nil to not be nil.'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[nil]'>
      <verdict method='verdict_refute_nil?' outcome='failed' id='1' message='Message 1'>
        <actual class='NilClass' value='nil'/>
        <exception class='Minitest::Assertion' message='Expected nil to not be nil.'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
</log>
